pset-functions.js

```diff
@@ -208,6 +338,32 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(string) {
+  //split string by word; second string has a comma after shark,
+  let words = string.toLowerCase().replace(',',' ').split(' ')
```

Comment:
Really well written code! You are taking advantage of method chaining to reduce code and using a for loop to push key/value pairs into an empty object through if/else blocks; really well done!

Great job throughout this homework; keep up the good work :)